gapis/api/vulkan/extensions/ext_line_rasterization.api

```diff
@@ -64,7 +64,7 @@ enum VkLineRasterizationModeEXT: u32 {
 @extension("VK_EXT_line_rasterization")
 class VkPhysicalDeviceLineRasterizationFeaturesEXT {
   VkStructureType sType
-  const void*     pNext
+  void*           pNext
```

Comment:
This change was wrong, but I understand why you did it. Cleaning up the `const void* pNext` vs `void* pNext` usage throughout our API files is on my TODO list.